### ISSUE TITLE
Die Zotero-Tags werden in allen Einträgen nicht richtig angezeigt (heute morgen war es noch okay)

### ISSUE THREAD

**Issue by SusanneHummel** [BetaMasaheft/Dillmann#283]
z.B.
http://betamasaheft.eu/Dillmann/lemma/L46e88b26281e44929a4e2c07f97f456a

**Comment by PietroLiuzzo** [comment]
Was ist nicht richtig?

**Comment by SusanneHummel** [comment]
Bei mir sieht es so aus:
xxiib ;   61a l. 25, 61b l. 1 (ed.)

also nur die Seitenzahlen - ohne "Bezold 1905"

**Comment by PietroLiuzzo** [comment]
Ich kann nicht dieses Fehler wiederholen. bitte probiermal die Seite neu zu laden.

**Comment by SusanneHummel** [comment]
Die App funktioniert bei mir allerdings wieder sehr sehr langsam - vielleicht deshalb?

**Comment by PietroLiuzzo** [comment]
Ja, jedesmal ich muss etwas direkt da machen, merkt man das, aber ich muss trotzdem...

**Comment by SusanneHummel** [comment]
denn lag es wohl daran; jetz geht es wieder. Dauert aber lange.
